src/foam/u2/Element.js

```diff
@@ -672,7 +672,7 @@ foam.CLASS({
   constants: [
     {
       name: 'CSS_CLASSNAME_PATTERN',
-      factory: function() { return /^[a-z_-][a-z\d_-]*$/i; }
```

Comment:
added whitespace as an possible character for class name. was getting an error otherwise

Reply:
Not a valid class name, but probably someone was setting two classnames at once. Can you detect when there's a whitespace a the do a 'debugger' so we can find who is doing that?

Reply:
sure